remedy.lic

```diff
@@ -56,6 +56,7 @@ class Remedy
   end
 
   def create_item
+    stow_hands
```

Comment:
Added. Thanks for the catch!

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

On 09/07/2016 01:30 PM, Robert Herbig wrote:
>
> In remedy.lic 
> <https://github.com/rpherbig/dr-scripts/pull/754#discussion_r77868549>:
>
> > @@ -56,6 +56,7 @@ class Remedy
> >    end
> >
> >    def create_item
> > +    stow_hands
>
> You need to put |common-items| in the |custom_require| call; and put 
> |include DRCI| at the top.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/rpherbig/dr-scripts/pull/754/files/35dd3fb4122f69d81ab64784878ceffe297007ad#r77868549>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AU8XXGnyvQwdIO1KLsjbmFK-ufnmuUWtks5qnvTDgaJpZM4J3E5a>.
>